### ISSUE TITLE
Allow the user to set a country

### ISSUE THREAD

**Issue by nightsh** [tenders-exposed/elvis-ember#85]
Dropdown similar to the query builder, just with one selectable option.

Not urgent, also blocked by the backend at the moment. Has no side effects in the rest of the app as far as I can tell.

**Comment by nightsh** [comment]
Blocked by tenders-exposed/elvis-backend#19

**Comment by zufanka** [comment]
@georgiana-b could you please add this issue on the new backend?

**Comment by victor-nitu** [comment]
Won't fix.
